.github/workflows/get-environment.yml

```diff
@@ -29,12 +29,6 @@ jobs:
 
       - id: get_environment
         run: |
-          if [[ -z "$GITHUB_HEAD_REF" ]]; then
-            BRANCHNAME="$GITHUB_REF_NAME"
-          else
-            BRANCHNAME="$GITHUB_HEAD_REF"
-          fi
-
```

Comment:
Kindly, could you restore this part ?
Until then, $BRANCHNAME will remain empty in the case below

Reply:
Done :)